flutter-studio/src/io/flutter/FlutterStudioStartupActivity.java

```diff
@@ -30,5 +31,6 @@ public void runActivity(@NotNull Project project) {
     // The IntelliJ version of this action spawns a new process for Android Studio.
     // Since we're already running Android Studio we want to simply open the project in the current process.
     replaceAction("flutter.androidstudio.open", new OpenAndroidModule());
+    replaceAction("ShowProjectStructureSettings", new FlutterShowStructureSettingsAction());
```

Comment:
Can we only do this replace if the given `project` is a Flutter project? We have some utils for detecting this, that look inside the pubspec.yaml file.